src/pages/profile/VerifyAirbnb.js

```diff
@@ -27,56 +33,76 @@ class VerifyAirbnb extends Component {
             defaultMessage={ 'Verify your Airbnb account' }
           />
         </h2>
-        { this.state.error == '' ? this.renderNormalFlow() : this.renderErrorMessage(this.state.error) }
+        <div className="general-error">{this.state.generalErrors.length > 0 ? this.state.generalErrors.join(' ') : ''}</div>
```

Comment:
personally, I like to hide the whole div if it has no content in case it has padding or something (now or in future styling revisions) that could affect surrounding elements and be confusing to debug